tests/plotting/test_plotting.py

```diff
@@ -4066,7 +4066,7 @@ def _generate_direction_object_functions() -> List[Tuple[str, FunctionType]]:
         if name[0].isupper() and (_has_param(func, 'direction') or _has_param(func, 'normal'))
     }
 
```

Comment:
This is the key line needed to add the new test case
```suggestion
functions['Capsule_legacy'] = functions['Capsule']
```